tests/test_plugin_flask_blueprint.py

```diff
@@ -48,7 +50,7 @@ def ping():
     """summary
 
     description"""
-    return jsonify(msg="pong")
+    return {"msg": "pong"}
```

Comment:
This is a bit edge casy but would actually support doing a second validation round .. The problem with this test case is that the endpoint expects a root model response (not supported previously) and returns a Flask response object.

Because we now validate root models, we capture a ValidationError on line 244 of the Flask plugin and return it later on. If we want to continue supporting this particular use case, we could skip line 216-250 when `isinstance(resp, flask.Response) is True` (then rely on the second validation in that case: lines 254-265).

I feel this is a small regression, but leaving the decision up to you @kemingy

Reply:
I think we need to check `isinstance(resp, flask.Response)`. This is a very common use case for flask.